{
  "FaultLabel": "Resolved power cut",
  "Type": "ResolvedFault",
  "Icon": "icon-success",
  "ShowViewOnMapCTA": false,
  "UnderReview": false,
  "AdditionalFaultInfo": "Good news! The electricity in your area has been restored. The power cut was caused by an unexpected incident on our underground cable that provides your supply.",
  "CTAs": [],
  "faultNumber": "60505429",
  "multipleFaultNumbers": null,
  "date": "2021-03-06T18:37:53",
  "region": "Dalton-In-Furness",
  "faultType": "Resolved Fault",
  "faultStatus": "Resolved",
  "previousFaultStatus": "Resolved",
  "consumersOff": 10,
  "estimatedTimeOfRestoration": "2021-03-07T18:00:00",
  "estimatedTimeOfRestorationMajority": "2021-03-06T18:37:53",
  "information": "",
  "actualTimeOfRestoration": "2021-03-06T18:37:53",
  "outageCentrePoint": {
    "lat": 54.157419,
    "lng": -3.186347
  },
  "outageLocations": null,
  "AffectedPostcodes": " LA15 8BP, LA15 8BS, LA15 8BT, LA15 8BU, LA15 8DE",
  "EtrMUnderReview": true,
  "WebTMSFaultType": "U/GM"
}